meiki_ui/src/components/app/toolbar/AppToolbar.svelte

```diff
@@ -12,8 +12,9 @@
     import UserIcon from "$cmp/icons/UserIcon.svelte"
     import ToolBarAction from "$cmp/toolbar/Action.svelte"
     import ToolbarSyncIndicator from "$cmp/app/toolbar/SyncIndicator.svelte"
+    import EditableLabel from "$cmp/EditableLabel.svelte"
 
-    export let title: string
+    export let title: string = "hello"
```

Comment:
Why Hello?

Reply:
I'm assuming this shouldn't ever be shown, but still something better than hello is  better?

Reply:
yep, will remove